### PR TITLE
docs(domain layer): update domain layer schema

### PR DESCRIPTION
**Pull request by warren830**
update domain layer schema for both tables and picture

### CODE REVIEW

File: docs/DataModels/DevLakeDomainLayerSchema.md [hunk 1]
```diff
@@ -428,80 +429,80 @@ Events of pull requests.
...
-| `branch`     | varchar  | 255       |                                                                 |         |
-| `repo_url`   | varchar  | 255       |                                                                 |         |
-| `id`         | varchar  | 255       | This key is generated based on details from the original plugin | PK      |
-| `created_at` | datetime | 3         |                                                                 |         |
-| `updated_at` | datetime | 3         |                                                                 |         |
-
+| **field** | **type** | **length** | **description**                                                 | **key** |
+| :-------- | :-------- | :-------- |:----------------------------------------------------------------| :-------- |
+| `id` | varchar | 255) | This key is generated based on details from the original plugin | PK |
+| `created_at` | datetime | 3)                             | Devlake common.NoPKModel                                        |  |
```
Comment (Startrekzky) [inline docs/DataModels/DevLakeDomainLayerSchema.md]: why is there a `) ` after every length, such as `3)`, `255)`? Can we remove them?

File: docs/DataModels/DevLakeDomainLayerSchema.md [hunk 2]
```diff
@@ -428,80 +429,80 @@ Events of pull requests.
...
+| `_raw_data_remark` | longtext | Devlake common.NoPKModel.RawDataOrigin              |                                                                 |  |
+| `name` | varchar | 255) |                                                                 |  |
+| `result` | varchar | 100)                           | The result of this task                                         |  |
+| `status` | varchar | 100)                           | The status of this task                                         |  |
+| `type` | varchar | 100)                           | To indicate if this is a DEPLOYMENT                             |  |
+| `duration_sec` | bigint unsigned |                                | how long does this task take                                    |  |
+| `started_date` | datetime | 3)                             | when did this task start                                        |  |
+| `finished_date` | datetime | 3)                             | when did this task finish                                       |  |
+| `environment` | varchar | 255)                           | To indicate if this is PRODUCTION(dora)                         |  |
```
Comment (Startrekzky) [inline docs/DataModels/DevLakeDomainLayerSchema.md]: Can we change it to `To indicate the environment in which the task is running`?

File: docs/DataModels/DevLakeDomainLayerSchema.md [hunk 3]
```diff
@@ -428,80 +429,80 @@ Events of pull requests.
...
+| `name` | varchar | 255)                           |                                                                 |  |
+| `pipeline_id` | varchar | 255)                           | The id of pipeline                                              |  |
+| `result` | varchar | 100)                           | The result of this task                                         |  |
+| `status` | varchar | 100)                           | The status of this task                                         |  |
+| `type` | varchar | 100)                           | To indicate if this is a DEPLOYMENT                             |  |
+| `duration_sec` | bigint unsigned |                                | how long does this task take                                    |  |
+| `started_date` | datetime | 3)                             | when did this task start                                        |  |
+| `finished_date` | datetime | 3)                             | when did this task finish                                       |  |
+| `environment` | varchar | 255)                           | To indicate if this is PRODUCTION(dora)                         |  |
```
Comment (Startrekzky) [inline docs/DataModels/DevLakeDomainLayerSchema.md]: Same as above

**Review by Startrekzky — CHANGES_REQUESTED** [review]

File: docs/DataModels/DevLakeDomainLayerSchema.md (hunk 1 above)
Comment (warren830) [reply docs/DataModels/DevLakeDomainLayerSchema.md]: > why is there a `) ` after every length, such as `3)`, `255)`? Can we remove them?

my bad, fixed

File: docs/DataModels/DevLakeDomainLayerSchema.md (hunk 2 above)
Comment (warren830) [reply docs/DataModels/DevLakeDomainLayerSchema.md]: > To indicate the environment in which the task is running

fixed

File: docs/DataModels/DevLakeDomainLayerSchema.md (hunk 3 above)
Comment (warren830) [reply docs/DataModels/DevLakeDomainLayerSchema.md]: > Same as above

fixed

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**